pandas/tests/indexes/multi/test_setops.py

```diff
@@ -692,6 +692,43 @@ def test_intersection_lexsort_depth(levels1, levels2, codes1, codes2, names):
     assert mi_int._lexsort_depth == 2
 
 
+@pytest.mark.parametrize(
+    "a",
+    [pd.Categorical(["a", "b"], categories=["a", "b"]), ["a", "b"]],
+)
+@pytest.mark.parametrize(
+    "b",
+    [
+        pd.Categorical(["a", "b"], categories=["b", "a"]),
```

Comment:
Is one of these supposed to have `ordered=True` like in the original issue? Otherwise both `b` params are the same